config/routes.rb

```diff
@@ -120,8 +121,6 @@
   get "/pages/*id" => "pages#show", :as => :page, :format => false
 
   # Authentication
-  get "auth/oauth2/callback" => "auth0#callback"
-  get "auth/failure" => "auth0#failure"
   get "sign_out" => "application#sign_out"
```

Comment:
I think we can delete this route too?

Reply:
Will do, thanks

Reply:
@CristinaRO note: this is in use in [`_navigation.html.haml`](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/ebaea76e659dadcfe298f9f9df36ec9e194d9667/app/views/shared/_navigation.html.haml#L30) on the `li`; we'll need to change this to the `path` on the link below it.